src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/media/VideoComponentObject.java

```diff
@@ -82,13 +82,13 @@ public void verifyVideoAnyclipEmbed() {
 
     WebElement object = container.findElement(By.tagName("object"));
     wait.forElementVisible(object);
-    Assertion.assertStringContains(getVideoPlayerObject().getAttribute("value"), object.getAttribute("id")
-    );
-    PageObjectLogging.log("verifyVideoAnyclipEmbed", "Anyclip video is embedded", true);
+    Assertion.assertStringContains(getVideoPlayerObject().getAttribute("value"),
+        object.getAttribute("id"));
+    LOG.success("verifyVideoAnyclipEmbed", "Anyclip video is embedded");
   }
 
   public void verifyVideoAutoplay(String providerName, boolean status) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "verifyVideoAutoplay" is 25 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)